Document Astro component import via `Astro.glob` (#2028)
withastro/docs: src/pages/en/guides/imports.md

</div>
```

> Imported Astro components via `Astro.glob` are of type [`AstroInstance`](/en/reference/api-reference/#astro-files), so we can access our components under the `default` property.

Reviewer: We usually use “You” rather than “We” when talking about what a user can do. A youser!

I’ve tried to rework this here:

```suggestion
Astro components imported using `Astro.glob` are of type [`AstroInstance`](/en/reference/api-reference/#astro-files). You can render each component instance using its `default` property:
```

```astro title="src/pages/component-library.astro" del={8} ins={9} ins=".default"
---
// imports all files that end with `.astro` in `./src/components/`
const components = await Astro.glob('../components/*.astro');
---
<!-- Display all of our components -->
{components.map((component) => (
  <div>

>     <component size={24} /> <!-- ❌ incorrect !-->

Author: Got it !
